### ISSUE TITLE
throughput in progress bar

### ISSUE THREAD

**Issue by roywei** [ultralytics/yolov5#10342]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi from recent update we found the throughput info is missing in progress bar, any idea why it's removed? it's quite useful for benchmarking. Thanks!
it's used to be like: with 5.26 it/s info
```

        0/2      8.36G     0.1027     0.1332    0.09538          6        640: 100%|██████████| 463/463 [01:29<00:00,  5.19it/s]
        1/2      8.36G    0.08447    0.08337    0.08307          6        640: 100%|██████████| 463/463 [01:28<00:00,  5.26it/s]
        2/2      8.36G    0.07142     0.0816    0.07307          3        640: 100%|██████████| 463/463 [01:28<00:00,  5.23it/s]
```

now it's
```
[1,0]<stderr>:      Epoch    GPU_mem   box_loss   obj_loss   cls_loss  Instances       Size
       0/99      14.3G      0.105     0.2458    0.09572        118        640: 100%|██████████| 58/58 00:31[1,0]<stderr>:
[1,0]<stderr>:
[1,0]<stderr>:      Epoch    GPU_mem   box_loss   obj_loss   cls_loss  Instances       Size
       1/99      14.3G     0.1005     0.0963    0.09465        187        640: 100%|██████████| 58/58 00:22[1,0]<stderr>:
[1,0]<stderr>:
[1,0]<stderr>:      Epoch    GPU_mem   box_loss   obj_loss   cls_loss  Instances       Size
       2/99      14.3G    0.09316    0.08498    0.08702        206        640: 100%|██████████| 58/58 00:22[1,0]<stderr>:
[1,0]<stderr>:
[1,0]<stderr>:      Epoch    GPU_mem   box_loss   obj_loss   cls_loss  Instances       Size
       3/99      14.3G    0.08193    0.08364    0.07913        148        640: 100%|██████████| 58/58 00:24[1,0]<stderr>:
[1,0]<stderr>:
[1,0]<stderr>:      Epoch    GPU_mem   box_loss   obj_loss   cls_loss  Instances       Size
       4/99      14.3G    0.07587    0.08373    0.07301        156        640: 100%|██████████| 58/58 00:24[1,0]<stderr>:
```

**Comment by glenn-jocher** [comment]
@roywei can you try out PR #10343 and see if this works better?

**Comment by glenn-jocher** [comment]
@roywei good news 😃! Your original issue may now be fixed ✅ in PR #10343. 

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
